spyql/processor.py

```diff
@@ -254,9 +273,20 @@ def eval_clause(self, clause, clause_exprs, mode="eval"):
         if not clause_exprs:
             return
         cmd = eval if mode == "eval" else exec
+        # pprint(self.vars)
         try:
-            return cmd(clause_exprs, self.vars, self.vars)
+            if self.interactive:
+                try:
+                    return cmd(clause_exprs, self.vars, {"data": self.vars["_values"]})
+                except:
+                    # when there is a FROM given but it is a python object
+                    # SELECT * FROM [1, 2, 3, 4]
+                    return cmd(clause_exprs, self.vars, self.vars)
+            else:
+                return cmd(clause_exprs, self.vars, self.vars)
+    
```

Comment:
I think if we revert this change everything should work as expected.
Regarding the `data` alias, we can add it as a translation in the constructor of the`PythonExprProcessor`, e.g.

   `self.translations["data"] = "_values"`

This will result in every reference to `data` being replaced by a reference to `_values`.

Reply:
Haha, one who creates it definitely knows their code better 😂. Struggled hard on this one, adding `self.translations["data"] = "_values"` in `PythonExprProcessor.__init__` works! 🙏